### PR TITLE
[ACCEL] changes cohort management dashboard tab translation method

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:** changes cohort management dashboard tab translation method

**Youtrack:** [ticket](https://youtrack.raccoongang.com/issue/ACCEL-527)

### CODE REVIEW

File: lms/djangoapps/cohort_management_dashboard/tab.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_noop as _
```
Comment (idegtiarov) [inline lms/djangoapps/cohort_management_dashboard/tab.py]: @Golub-Sergey Please explain why do we need this change?

Comment (Golub-Sergey) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: for most of tabs uses `ugettext_noop`, if we use this method cohort dashboard tab translation changes correctly

Comment (Golub-Sergey) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: when we use `ugettex` method it translate text of tab once and then doesn't translate, if we use `ugettext_noop` method it translates always when we call it. `ugettext_noop` uses in all tabs

Comment (idegtiarov) [reply lms/djangoapps/cohort_management_dashboard/tab.py]: Thanks for the explanation!

**Pull request merged**